sdk/maps/azure-maps-service/samples/alias.py

```diff
@@ -0,0 +1,63 @@
+# These examples are ingested by the documentation system, and are
+# displayed in the SDK reference documentation. When editing these
+# example snippets, take into consideration how this might affect
+# the readability and usability of the reference documentation.
+import argparse
+
+from azure.maps.service.operations import AliasOperations
+from azure.core.exceptions import HttpResponseError
+from typing import List
+from common.common import create_maps_client
+
+
+def create(alias: AliasOperations):
```

Comment:
We should not have examples that takes operations classes. All examples should use the `client` class directly.